src/main/java/com/commercetools/project/sync/model/response/ReferenceIdKey.java

```diff
@@ -0,0 +1,45 @@
+package com.commercetools.project.sync.model.response;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public class ReferenceIdKey {
+  private final String id;
+  private final String key;
+
+  @JsonCreator
+  public ReferenceIdKey(
+      @JsonProperty("id") @Nonnull final String id,
+      @JsonProperty("key") @Nullable final String key) {
+    this.id = id;
+    this.key = key;
+  }
+
+  public String getId() {
+    return id;
+  }
+
+  public String getKey() {
+    return key;
+  }
+
+  @Override
+  public boolean equals(final Object other) {
+    if (this == other) {
+      return true;
+    }
+    if (!(other instanceof ReferenceIdKey)) {
+      return false;
+    }
+    final ReferenceIdKey that = (ReferenceIdKey) other;
+    return Objects.equals(getId(), that.getId()) && Objects.equals(getKey(), that.getKey());
```

Comment:
Hmm, not sure why you think it's better?

Here's my two cents why I prefer `Objects#equals`:

1. It is native `java.utils` not an external dependency like `EqualsBuilder` which is from `apache.commons`.
2. `EqualsBuilder` instantiates a new object while `Objects#equals` doesn't.
2. I think it is easier to read than having "append".
2. I think it is more common/known since it's native.

Would be curious why you think is better?

Reply:
I thought we already have the dependency. Yes, in fact your usage would be better. I was curious about the lengthy statement, but I think this would be simplest form we can do the equal comparison. :)